Backend/Database/forebar_table.sql

```diff
@@ -15,4 +14,4 @@ CREATE TABLE IF NOT EXISTS partSubmission.Forebar (
 )
 ENGINE=InnoDB
```

Comment:
We can remove these last 3 lines in any of the SQL files; these lines don't need to be in our files even though DBeaver gives it to us:
```
ENGINE=InnoDB
DEFAULT CHARSET=utf8mb4
COLLATE=utf8mb4_0900_ai_ci;
```